### ISSUE TITLE
Geolocation panel: map image disappears when panel is stacked horizontally

### ISSUE THREAD

**Issue by guillaumejenkins** [Microsoft/cordova-simulate#9]
Tested in Chrome. When the plugin UI window is narrow and the Geolocation panel is stacked vertically with the other panels, everything is fine:

![image](https://cloud.githubusercontent.com/assets/14299377/14896324/09067f72-0d32-11e6-92c7-febd4018ddbb.png)

However, when you resize the page to be wider and the Geolocation panel is moved to the right of other panels, the map background disappears:

![image](https://cloud.githubusercontent.com/assets/14299377/14896344/1f0df444-0d32-11e6-819f-2275a7de824f.png)

**Comment by TimBarham** [comment]
This is caused by a Chrome bug which we reported months ago and unfortunately still isn't fixed. Basically, in Chrome, any element that is the child of an element with overflow set to hidden will not be displayed in anything other than the first column when using a multi-column layout. I worked around this where I could (though I think my work-around is the cause of the #10 that @albertinad filed). Unfortunately the way the map works it really requires overflow set to hidden, so I haven't come up with a work-around there. One solution would be to simply force the geolocation panel to always be the first panel (so it is always in the first column).
